docs/upgrade-testing.md

```diff
@@ -0,0 +1,58 @@
+## Upgrade testing 
+Follow below step for manual upgrade testing 
+
+1. Set environment variables to overwrite values in Makefile
+
+```
+IMAGE_OWNER ?= opendatahub
+VERSION ?= 2.4.0
+```
+
+2. Add `replaces` proprty in `opendatahub-operator.clusterserviceversion.yaml` file under `config/manifests/bases` and add version which you would like to upgrade with next version
+
+```
+replaces: opendatahub-operator.v2.4.0
+```
+
+3. Build and push docker container image
+
+```
+make image
+```
+
+4. Build bundle image 
+
+```
+make bundle-build
+```
+
+5. Push bundle image into registry
+
+```
+make bundle-push
+```
+
+6. Build catalog source image 
+
+```
+make catalog-build
+```
+
+7. Push catalog source image into registry
+
+```
+make catalog-push
+```
+### Cluster
```

Comment:
this whole section need more information.
plain words wont help to know what each step is for.